src/components/FlexibleForm/Control.js

```diff
@@ -61,10 +66,20 @@ export const Control = memo(props => {
   const isFragment = (isEditable && controlType === 'Fragment') || (!isEditable && staticControlType === 'Fragment');
 
   const getOptions = options => {
-    return options.map(option => ({
-      value: option.value,
-      label: intl.formatMessage({ id: option.label }),
-    }));
+    return options.map(option => {
+      const lbl = option.label;
```

Comment:
```suggestion
      const { label } = option;
```
In my opinion, the variables like `lbl` or `nsName` can be unreadable at some point. Now, it does not pose a problem as the methods are small. Overall, I would prefer readability over brevity.